modules/accredited_representative_portal/db/migrate/20241223094500_make_poa_request_and_form_adjustments.rb

```diff
@@ -0,0 +1,24 @@
+class MakePoaRequestAndFormAdjustments < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
```

Comment:
Not sure if this is necessary, since these tables should have 0 records in all fields, do we know why it's in here or just out of habit?

Reply:
probably just habit.